.circleci/config.yml

```diff
@@ -739,46 +738,3 @@ workflows:
           test_apk: *salesforceReact-apk-path
           api_level: 25
 
-  test-api-23:
```

Comment:
Side note:  This file has no business being 700+ lines long.  I will come back to this and fix it up with matrix jobs and/or CircleCI's new monorepo support.